### ISSUE TITLE
Add ability to consume from multiple queues/exchanges

### ISSUE THREAD

**Issue by markstory** [markstory/go-consumer#1]
It would be nice to allow a single consumer to read messages from more than one queue/exchange. Perhaps the configuration could look like:

```
    [connection]
    host = localhost
    virtual_host = '/'
    user = guest
    password = guest
    ssl = False

    [exchange-app]
    name = app
    type = direct
    durable = True
    auto_delete = False

    [queue-app]
    name = app
    durable = True
    auto_delete = True
    exclusive = False
    routing_key = app-event

    [exchange-backend]
    name = backend
    type = direct
    durable = True
    auto_delete = False

    [queue-backend]
    name = backend
    durable = True
    auto_delete = True
    exclusive = False
    routing_key = backend-event
```

When multiple queues are being bound, each `exchange` and `queue` section should be suffixed with the same value. This defines the binding between the exchange and queue.

**Comment by markstory** [comment]
This is mostly implemented outside of some error handling as of bf7667e857dba79c8e651287e54c5604a09a2511

**Comment by markstory** [comment]
Error handling is now done too.
